FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/NDayMultiComputable.java

```diff
@@ -33,14 +33,32 @@ public float[] multiCompute(float[] values) {
         values = stepOneCompute.multiCompute(values);
 
         for(float day : days) {
-            int timeStep = (int) config.getDuration().toHours();
-            int timeStepDay = 24 / timeStep;
-            results[i] = values[timeStepDay * (int) day];
+            int timeStep = (int) config.getDuration().getSeconds();
```

Comment:
Should a similar change be made in the above "compute" method?

Reply:
Actually, yes this needs to be updated as well.  Found a bug as a result too so added an additional conditional statement when the accumulated value is the last value in the array.